proposals/description/eth_lbp.ts

```diff
@@ -0,0 +1,64 @@
+import { ProposalDescription } from '@custom-types/types';
+
+const eth_lbp: ProposalDescription = {
+  title: 'FIP-110: Reinforce PCV via ETH LBP and tighter spread',
+  commands: [
+    {
+      target: 'pcvGuardianNew',
+      values: '0',
+      method: 'setSafeAddresses(address[])',
```

Comment:
Metacomment - there is a singular version of this function that is easier to call